modules/services/blueman-applet.nix

```diff
@@ -24,6 +24,19 @@ in
       };
 
       package = lib.mkPackageOption pkgs "blueman" { };
+
+      systemdTargets = lib.mkOption {
+        type = with lib.types; listOf str;
+        default = [ "graphical-session.target" ];
+        example = [ "sway-session.target" ];
+        description = ''
+          The systemd targets that will automatically start the cliphist service.
+
+          When setting this value to `["sway-session.target"]`,
+          make sure to also enable {option}`wayland.windowManager.sway.systemd.enable`,
+          otherwise the service may never be started.
```

Comment:
I notice this has been copy pasted around the codebase, a lot. I think it's unnecessary and can be cleaned up in a treewide effort. But, we can leave it, for now.